### DISCUSSION TITLE
For some reason today I get different images using yesterday's prompts. What can be the case?

### DISCUSSION THREAD

**Discussion by Bra2ha** [lllyasviel/stable-diffusion-webui-forge#2246] [Q&A]
I didn't change the hardware, didn't update the drivers. 
![3453453453453462417-796772140](https://github.com/user-attachments/assets/d3ee7363-a985-4587-9fa0-bf5328a84806)
Yesterday's image
![3453453453453462530-796772140](https://github.com/user-attachments/assets/f5d511d8-a983-4483-8a02-873ce1bc5640)
Today's image

**Comment by Bra2ha** [comment]
Also, I get different results each time I press Generate, with locked seed. 
![3453453453453462532-2094465471](https://github.com/user-attachments/assets/e5f9523d-5cb2-47ea-899f-e961b3bdbc54)
![3453453453453462531-2094465471](https://github.com/user-attachments/assets/40c27302-32c5-4c7a-843a-15c2d8c0cb94)

**Comment by iqddd** [comment]
Same. Same seed, prompt etc - different results.

**Comment by iqddd** [comment]
Looks like problem occurs only with “Flux realistic” sampler.

**Comment by lllyasviel** [comment]
i seem to be able to reproduce. will take a look next time when i am on forge

**Comment by ReppiksProductions** [comment]
I'm having the same issue with the "Flux Realistic" sample.  I didn't notice for a long time because I've been doing some testing on my LORA.  I thought the difference in my epochs was causing drastic changes to the seed LMAO.  Now I'm going to have to do hours of comparison generations again smh, but at least it will be easier to compare.
Are we sure this isn't the expected result when using "Flux Realistic"?

**Reply by iqddd** [reply]
We don't need non-deterministic samplers. That's for sure. I think there's a non-seeded random being used somewhere.

**Comment by djasil** [comment]
@lllyasviel I think I found the culprit commit: [1b44ae2](https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/1b44ae22e3996b8af048f268bd076702b296b469)

I just restored the previous GOOGLE_BLOCKLY_COMMIT_HASH in modules/launch_utils.py:
1e98997c7fedaf5106af9849b6f50ebe5c4408f1 -> **bf36e6fd3750a081f44209ba4f645adb598f7e37**

I don't know what this hash is and how it is being used, but this change fixed the issue on my side with Flux Realistic (reproduced the same old gens with LoRA every time).

**Reply by Bra2ha** [reply]
It works 👍 Thank you

**Reply by tristanwqy** [reply]
> @lllyasviel I think I found the culprit commit: [1b44ae2](https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/1b44ae22e3996b8af048f268bd076702b296b469)
> 
> I just restored the previous GOOGLE_BLOCKLY_COMMIT_HASH in modules/launch_utils.py: 1e98997c7fedaf5106af9849b6f50ebe5c4408f1 -> **bf36e6fd3750a081f44209ba4f645adb598f7e37**
> 
> I don't know what this hash is and how it is being used, but this change fixed the issue on my side with Flux Realistic (reproduced the same old gens with LoRA every time).

that saved my life!

**Reply by grvdggr** [reply]
Thanks for sharing, really appreciate it. =)

**Reply by grvdggr** [reply]
Unfortunately, [Forge] Flux Realistic (1.5x Slow) disappeared when I changed the hash and that's the best one to use along with DDIM as Scheduler according to my tests.
I changed the hash back and will wait for a solution that does not delete [Forge] Flux Realistic (1.5x Slow).

**Comment by iqddd** [comment]
What are those two modules from ‘~/stable-diffusion-webui-forge/repositories/google_blockly_prototypes/forge’ anyway?
additional_samplers.pyz
icl_v2.pyz
The code is compressed and obfuscated.
They recently added the ‘skip-google-blockly’ startup flag.
```python
parser.add_argument("--skip-google-blockly", action='store_true', help="launch.py argument: do not initialize google blockly modules")
```

**Reply by shaun-ba** [reply]
> google_blockly_prototype

quick google: https://github.com/lllyasviel/google_blockly_prototypes

> Some of our projects will begin to use Google Blockly as a prototyping tool for certain developments. This approach is aimed at enhancing the speed and flexibility of experimental feature creation.

**Reply by usigg3** [reply]
I put --skip-google-blockly in webui-user.bat and that seems to fix the problem. Not sure if it has other effects... Can't see any so far.

**Reply by nektosu** [reply]
If put --skip-google-blockly in webui-user.bat
[Forge] Flux Realistic
disappears from the list

**Reply by usigg3** [reply]
For me it's still there.

**Comment by kalle07** [comment]
same here ...
any solution ?
that "--skip-google-blockly" can only be a fast fix

**Reply by danilomaiaweb** [reply]
This solution work for me: https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/2246#discussioncomment-11385756

**Reply by kalle07** [reply]
thx i use swarmUI -> forge are OUT   ;)
even better results in image quality and loras in any case... cause its based on comfy  ;)

**Reply by grvdggr** [reply]
Nothing in Swarm beats the Flux Realistic Samplers and Loras didn't work as well in Forge.
Hence why I got back to Forge.

**Reply by kalle07** [reply]
for me it is and Flux Realistic is nearly .
"DPM++ 2S Ancestral 2nd Order"

.. and you can play with compfy  :)

**Reply by iqddd** [reply]
DPM++ 2S Ancestral 2nd Order = DPM++ 2S a

**Reply by kalle07** [reply]
https://github.com/lllyasviel/stable-diffusion-webui-forge/pull/2151

**Reply by iqddd** [reply]
Gosh, what a mess!

**Reply by iqddd** [reply]
> thx i use swarmUI -> forge are OUT ;)

Has lllyasviel said anything about the lack of updates on Forge?

**Reply by kalle07** [reply]
i dont know him ^^
but he has also a part of fooocus what has ended because mashb1t has left/paused
